samcli/lib/build/app_builder.py

```diff
@@ -777,6 +777,10 @@ def _get_build_options(
             )
             if working_directory:
                 options = {**options, "working_directory": convert_path_to_unix_path(working_directory)}
+
+        if language == "rust" and "Binary" in build_props:
```

Comment:
What is `"Binary" in build_props` for?

Reply:
It maps to the `--bin` flag of Cargo Lambda. It is to specify which binary to build in a project structure that contains multiple binaries (as defined in each `[[bin]]` section in Cargo.toml). 
As for SAM CLI, this can be useful for cases like multiple Lambda Functions, each pointing to a specific binary.

Reply:
Got it. I don't know enough to have a deeper opinion or thoughts. :)